forge/ee/db/views/FlowTemplate.js

```diff
@@ -11,12 +11,13 @@ module.exports = function (app) {
             icon: { type: 'string' },
             order: { type: 'number' },
             default: { type: 'boolean' },
+            flows: { type: 'array' },
             createdAt: { type: 'string' },
             updatedAt: { type: 'string' }
         }
     })
     function flowBlueprintSummary (blueprint) {
-        return {
+        const newVar = {
```

Comment:
why make it a const and not just return as before? Also, naming of this is a bit odd. Guessing this was a debug thing at some point?

Reply:
true, that was for debugging purposes, reverted!